retweet.go

```diff
@@ -51,3 +52,84 @@ func retweetsLookup(ctx context.Context, c *client, tweetID string, opt ...*Retw
 
 	return &retweetsLookup, nil
 }
+
+func postRetweet(ctx context.Context, c *client, userID string, tweetID string) (*PostRetweetResponse, error) {
+	if userID == "" {
+		return nil, errors.New("post retweet by userID: userID parameter is required")
```

Comment:
I think this error message isn't good.
This is because some of the error messages are duplicated.

Reply:
sorry, I'll change to `post retweet: userID parameter is required`